runware/types.py

```diff
@@ -314,6 +318,28 @@ def __post_init__(self):
                 }
             )
 
+    def validate_number_results(self):
+        if self.numberResults is None or not isinstance(self.numberResults,
+                                                        int) or self.numberResults < 1 or self.numberResults > 20:
+            raise ValueError(
```

Comment:
should we use IError here?

Reply:
@bghira I can use it , but we need to merge this one first: https://github.com/Runware/sdk-python/pull/43